spec/OBD/OBD.vspec

```diff
@@ -68,85 +75,99 @@
   description: PID 04 - Engine load in percent - 0 = no load, 100 = full load
 
 - CoolantTemperature:
-  datatype: float
+  datatype: int16
   type: sensor
   unit: celsius
+  min: -40
+  max: 215
   description: PID 05 - Coolant temperature
 
 - ShortTermFuelTrim1:
-  datatype: int8
+  datatype: float
```

Comment:
Here we lose precision if using int8. The OBD scaling from uint8 to real value is (100*X)/128-100, i.e. it offers somewhat better precision than an int8.